### PR TITLE
Search service integration

### PR DESCRIPTION
**Pull request by pega-roska**
Add support for configuring the chart to use the external search service.

### CODE REVIEW

File: charts/pega/templates/pega-environment-config.yaml [hunk 1]
```diff
@@ -32,12 +32,21 @@ data:
...
   # URL to connect to Elastic Search
   PEGA_SEARCH_URL: {{ .Values.pegasearch.externalURL }}
+{{ end }}
   # Whether to enable connecting to a cassandra cluster.  "true" for enabled, "false for disabled"
   CASSANDRA_CLUSTER: "{{ include "cassandraEnabled" . }}"
   # Comma separated list of cassandra hosts
   CASSANDRA_NODES: "{{ include "cassandraNodes" . }}"
   # Port to connect to cassandra with
   CASSANDRA_PORT: "{{ .Values.dds.port }}"
+  # Deployment ID for FNX services
```
Comment (misterdorito) [inline charts/pega/templates/pega-environment-config.yaml:50]: Does this belong here?

Comment (pega-roska) [reply charts/pega/templates/pega-environment-config.yaml]: Replaced with more generic phrasing

**Review by taz-pega-work — COMMENTED** [review]
I believe you can clarify this cluster of statements to be active voice and group action:
# Only attempt to generate this file IF the user has NOT supplied
# an external search URL or enabled the search service, which means we
# need to deploy Elasticsearch internally.
===>
The deployment generates this file to invoke Elasticsearch internally when the external search URL is blank (or the default value is left) AND no search service is enabled.

**Comment by pega-roska** [comment]
> I believe you can clarify this cluster of commented statements to be active voice and group the context clearly:
> Only attempt to generate this file IF the user has NOT supplied
> an external search URL or enabled the search service, which means we
> need to deploy Elasticsearch internally.
> ===>
> The deployment generates this file to invoke Elasticsearch internally when the external search URL is blank (or the default value is left) AND no search service is enabled.

This should be addressed, now.

**Review by misterdorito — APPROVED** [review]

**Review by pega-talba — APPROVED** [review]

**Pull request merged**